jobs/webcompat-kb/webcompat_kb/commands/render.py

```diff
@@ -0,0 +1,63 @@
+import argparse
```

Comment:
LGTM. Wonder if having some docs describing checkdata and render commands could be useful?